orchestra/static/orchestra/common/css/_common.scss

```diff
@@ -74,3 +74,124 @@ $main-font-stack: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helveti
      transform:rotate(360deg);
   }
 }
```

Comment:
does the time-tracking modal and each screen from the top right menu (e.g., account settings, projman, ...) still display properly?

Reply:
Yes! Sent you a video walkthrough offline to confirm.